jetty-client/src/main/java/org/eclipse/jetty/client/HttpDestination.java

```diff
@@ -326,10 +326,10 @@ private void process()
         }
     }
 
-    public boolean process(final Connection connection)
+    public boolean process(Connection connection)
     {
         HttpClient client = getHttpClient();
-        final HttpExchange exchange = getHttpExchanges().poll();
```

Comment:
These finals were required to get rid of  code style warnings.   Why are you taking them out again?
Even if we don't want them, don't make unrelated changes in a PR.

Reply:
What warnings? 
They are unnecessary and clutter the code, so while I touched the file I got rid of them.